prog/weavewait/main.go

```diff
@@ -1,17 +1,25 @@
 package main
 
 import (
+	"errors"
 	"fmt"
 	"os"
 	"os/exec"
+	"os/signal"
 	"syscall"
-
-	"github.com/weaveworks/weave/net"
+	"time"
 )
 
+var timeoutErr = errors.New("Container timed out waiting for ethwe interface")
```

Comment:
inline that. and make the description accurate. "Container timed out waiting for signal from proxy"?